code/modules/mob/living/carbon/human/life.dm

```diff
@@ -75,7 +75,10 @@
 	if(species.species_flags & NO_BREATHE)
 		return
 
-	if(losebreath <= 10)
+	if(pulledby?.grab_state >= GRAB_KILL)
+		Losebreath(1)
+		adjustOxyLoss(4)
+	else if(losebreath <= 10)
 		adjust_Losebreath(-1) //Since this happens before checking to take/heal oxyloss, a losebreath of 1 or less won't do anything.
 	else
 		set_Losebreath(10) //Any single hit is functionally capped - to keep someone suffocating, you need continued losebreath applications.
```

Comment:
```suggestion
	if(pulledby?.grab_state >= GRAB_KILL)
		Losebreath(1)
		adjustOxyLoss(4)
	else if(losebreath > 10)
		set_Losebreath(10) //Any single hit is functionally capped - to keep someone suffocating, you need continued losebreath applications.
	else if(losebreath > 0)
		adjust_Losebreath(-1) //Since this happens before checking to take/heal oxyloss, a losebreath of 1 or less won't do anything.
```
I got annoyed by this using extra adjust calls and pushing losebreath to negative lots uselessly.